modules/API/pages/built-in-endpoints.adoc

```diff
@@ -3441,6 +3441,144 @@ Response::
 | String ID of the query.
 |===
 
+
+=== Get information about a template query
+
+`GET /gsql/library?functionName=<package>.<package>.<template name>`
+
+This endpoint returns the query definition, query name, and query parameters for a template query.
```

Comment:
not `query definition`, it is template definition, you may omit `query`.
The same goes for everything else.